requirements/requirements_ansible.in

```diff
@@ -45,7 +45,6 @@ psutil==5.4.3    # same as AWX requirement
 setuptools==36.0.1
 pip==9.0.1
 # VMware
-psphere==0.5.2
```

Comment:
I tried to cull this list before, and if something in is the current list, it's probably because something prevented me from getting rid of it. It looks like psphere is still imported in the vmware script.

https://github.com/ansible/ansible/blob/a11f631ee414ba1048b04564cc901f1ff8b67479/contrib/inventory/vmware.py

However, our script is called `vmware_inventory.py` in plugins. The documentation in these scripts are not helping me understand what happened there, but it looks like maybe this really was just missed?

Reply:
Maybe @wenottingham's assertions at https://github.com/ansible/awx/pull/2506 are not actually true?

Reply:
Also, I'm mostly certain that the python vmware psphere SDK doesn’t support py3:

https://github.com/jkinred/psphere/issues/9

Reply:
yeah, this looks fine to me. It was planned to be removed with this PR. I don't anticipate any issues.

Reply:
Thankfully, it looks like the plugin is a continuation of `vmware_inventory.py` and not `vmware.py`, and also does not use psphere.

https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/inventory/vmware_vm_inventory.py

So we should be safe there as well.